dom.bs

```diff
@@ -5370,8 +5385,11 @@ these steps:
  <li><p>If <var>node</var> is a <a for=/>shadow root</a>, then <a>throw</a> a
  "{{HierarchyRequestError!!exception}}" {{DOMException}}.
 
- <li><p>If <var>node</var> is a {{DocumentFragment}} <a for=/>node</a> whose
- <a for=DocumentFragment>host</a> is non-null, then return.
+ <li>
+  <p>If <var>node</var> is a {{DocumentFragment}} <a for=/>node</a> and its
+  <a for=DocumentFragment>host</a> is non-null, then return <var>node</var>.
+
+  <p class=note>Unfortunately this does not throw for web compatibility.
```

Comment:
What does "does not throw for web compatibility" mean? Any one who employs `template.content` node adoption before will now face with breakage with the sudden change of "whose host is non-null, then return".

If we no longer guarantee old API evolves without breaking, how do I find some rule of thumbs what is dangerous and what is not? An example issue is here https://github.com/aurelia/aurelia/issues/1264

Reply:
> From reading the issue you linked it seems that implementations never matched the specification and some applications (such as yours) relied on that. 
That's not true. The implementations were according to the spec until #754, that inserted a rule about template's fragment for the first time. 
That rule returned undefined, there were some issues with that and the browsers didn't implemented (according to [this ](https://github.com/whatwg/dom/issues/813#issuecomment-570191640).
Then with #819 the spec changed again. 
On July webkit implemented the new rule. Before one month Safari went public with the new version of webkit and many applications broke on **production**.
And since then we are trying to figure out what's going on.

Reply:
So it sounds from the linked issues and comment like this change isn't web compatible as-is, right? Sounds like another approach is needed.

Reply:
Any updates on this?
Right now our application is not working on Safari on production. 
Webkit [says ](https://bugs.webkit.org/show_bug.cgi?id=246899) that this is intentional behavior change and we don't know how to proceed to fix our application...